src/Aspire.ProjectTemplates/templates/aspire-starter/AspireStarterApplication.1.AppHost/Program.cs

```diff
@@ -10,6 +10,7 @@
 #if UseRedisCache
     .WithReference(cache)
 #endif
-    .WithReference(apiService);
+    .WithReference(apiService)
+    .WithExternalHttpEndpoints();
```

Comment:
I would add it first not last.

Reply:
Curious why? is there some race?

Reply:
Not a race but this exposes all http/https endpoints defined when called. That's the more correct behavior for our default experience. Other http endpoints won't get exposed externally, which would blow up in ACA

Reply:
Will make the change, but not understanding how it affects the WithReference calls in this case.

Reply:
It doesn't. It's about future calls to `WithHttpEndpoint`. If they get made before `WithExternalHttpEndpoints` it'll likely explode on ACA.

Reply:
Sounds like another good case for an analyzer ;-)